app/assets/translations/appConstants/bebbo/ALsq.ts

```diff
@@ -389,10 +390,15 @@ const ALsq = {
               parentGender: "Gjinia",
               successOnboardingImport: "Të dhënat e fëmijës u importuan me sukses",
               updateImportText: "Ju lutemi të plotësoni të dhënat për të vazhduar",
-              importOnboardingText: "Ju mund të importoni të dhënat e fëmijës që keni eksportuar më parë",
-              OnboardingImportButton: "Importo tani",
+              importOnboardingText1: "Informacioni që fut në aplikacion ruhet në telefonin tuaj ose në hapësirën tuaj ruajtëse personale.",
+              importOnboardingText2: "Nëse keni përdorur Bebbo në një pajisje tjetër dhe keni eksportuar informacionin tuaj, mund ta ngarkoni skedarin në këtë pajisje tani.",
+              importOnboardingText: "Bebbo nuk mbledh të dhënat tuaja personale.",
+              OnboardingImportButton: "Ngarko tani ",
```

Comment:
Is space intentional?

Reply:
@Akhror, Evrim send me all string changes so I use same here. I think space is by mistake.